spec/models/user_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe User do 
+
+	let!(:user) do
+    User.create(name: 'Dan Wheeler', username: 'd9nny',
+								email: 'dan@google.com', password: 'test123', 
+								password_confirmation: 'test123')
+  end
```

Comment:
Lint/BlockAlignment: `end` at 9, 2 is not aligned with `let!(:user) do` at 5, 1.